ffcx/codegeneration/finite_element_template.py

```diff
@@ -20,42 +20,26 @@
   {reference_value_dimension}
 }}
 
-int evaluate_reference_basis_{factory_name}(double* restrict reference_values,
-                                            int num_points,
-                                            const double* restrict X)
-{{
-  {evaluate_reference_basis}
-}}
-
-int evaluate_reference_basis_derivatives_{factory_name}(double * restrict reference_values,
-                                          int order, int num_points,
-                                          const double * restrict X)
-{{
-  {evaluate_reference_basis_derivatives}
-}}
-
 int transform_reference_basis_derivatives_{factory_name}(
```

Comment:
Why is this method still in ufc?

Reply:
It's still used in dolfinx, but could possibly be more neatly replaced by implementing Piola mapping functionality in Basix

Reply:
I've made an issue: #299